task/fbc-validation/0.1/fbc-validation.yaml

```diff
@@ -121,27 +121,32 @@ spec:
         # copy content of conffolder to confdir - will be used in next task - related image check
         cp -r .$conffolder/* $(workspaces.workspace.path)/hacbs/$(context.task.name)/confdir
 
-        # We have totally 4 checks here currently
-        check_num=4
+        EXTRACT_DIR="/extracted_base_img"
+        mkdir "${EXTRACT_DIR}"
+        oc image extract ${BASE_IMAGE} "${EXTRACT_DIR}"
+        OPM_BINARIES="$(find ~+${EXTRACT_DIR} -type f -name opm)"
+        if [[ $(wc -l <<< "$temp") = "1" ]]; then
```

Comment:
Just out of curiosity, what does `<<< "$temp"` do? I tried it with different(random) name of the variable and it always returns 1.

Reply:
Is **$temp** defined natively?

Reply:
That was a leftover from writing the code.  I've replaced `$temp` with `${OPM_BINARIES}`

Reply:
That should also be not equal to one.  I've fixed that too